src/test/kotlin/blackjack/domain/card/CardTest.kt

```diff
@@ -0,0 +1,26 @@
+package blackjack.domain.card
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class CardTest : StringSpec({
+
+    "총 48장의 카드 목록을 반환한다" {
```

Comment:
```suggestion
    "총 52장의 카드 목록을 반환한다" {
```
블랙잭은 총 52장의 카드가 있어야합니다!

Reply:
10까지 포함되는군요! J, Q, K만 10으로 표현되는 줄로 이해했네요. 반영했습니다 :)
https://github.com/next-step/kotlin-blackjack/pull/465/commits/dffe4771aebab7acf8229b65081c1f00e3b02b63